packages/consumption/src/consumption/ConsumptionCoreErrors.ts

```diff
@@ -229,6 +229,13 @@ class Attributes {
         );
     }
 
+    public wrongRelationshipStatusToSetDeletionInfo() {
+        return new CoreError(
+            "error.consumption.attributes.wrongRelationshipStatusToSetDeletionInfo",
+            "In order to manually set the deletionInfo of an Attribute, the corresponding Relationship must be in status DeletionProposed."
```

Comment:
For consistency with other error messages.

```suggestion
            "In order to manually set the deletionInfo of an Attribute, the corresponding Relationship must be in status 'DeletionProposed'."
```